package.json

```diff
@@ -259,6 +259,7 @@
     "url-join": "5.0.0",
     "validate-npm-package-name": "6.0.1",
     "vuln-vects": "1.1.0",
+    "write-file-atomic": "6.0.0",
```

Comment:
Why is a regular fs write not ok? Is there some concurrency to be worried about?

Reply:
Okay, I've traded safe writes for one less dependency.